### PR TITLE
Reset disks on startup

### PR DESCRIPTION
**Pull request by kintel**
TODO:
- [x] Reset disks at start and end of disk game
- [x] Reset all views at _beginFirstGame()

### CODE REVIEW

File: app/src/views/disk-view.js [hunk 1]
```diff
@@ -46,7 +47,9 @@ export default class DiskView {
   }
 
   _handleChanges(changes) {
-    if (changes.hasOwnProperty('disk') && changes.disk.hasOwnProperty('level') || changes.currentGame === "disk") {
+    if (changes.currentGame === GAMES.HANDSHAKE || // Reset on startup
```
Comment (sunjay) [inline app/src/views/disk-view.js]: This adds a dependency on the game sequence and will break if we change it. Instead, just call reset disks from the constructor.

Comment (kintel) [inline app/src/views/disk-view.js]: I believe the constructor is called before serial is initialized. To fix this we'd first have to refactor the initialization sequence.

Comment (kintel) [inline app/src/views/disk-view.js]: I guess what I'd really want would be an initial state change on initialization

Comment (sunjay) [inline app/src/views/disk-view.js]: Well this is a very view specific requirement. I don't think there is an initial state change to put here. I suggest adding a public `reset()` method to all of the views and call that in start first game

**Comment by kintel** [comment]
@sunjay Try again now

**Comment by sunjay** [comment]
Looks good!

**Pull request merged**